DevCycle/DVCClient.swift

```diff
@@ -166,6 +158,26 @@ public class DVCClient {
         }
     }
 
+    func setupSSEConnection() {
```

Comment:
private?

Reply:
Yup, good point. I guess since this method isn't tested there's not point in it being Internal